components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1294,6 +1294,11 @@ void RewardsServiceImpl::RecoverWallet(const std::string& passPhrase) {
 }
 
 void RewardsServiceImpl::OnRecoverWallet(const ledger::type::Result result) {
+  // Fetch balance after recovering wallet in order to initiate P3A
+  // stats collection
+  FetchBalance(base::BindOnce(
+      [](ledger::type::Result result, ledger::type::BalancePtr balance) {}));
```

Comment:
Nit: would `base::DoNothing()` work here?

Reply:
Yes, thanks for mentioning - fixed.